custom_components/elegoo_printer/definitions.py

```
@@ -366,3 +379,34 @@ class ElegooPrinterLightEntityDescription(
     ),
 )
+
+PRINTER_FDM_BUTTONS: tuple[ElegooPrinterButtonEntityDescription, ...] = (
+    ElegooPrinterButtonEntityDescription(
+        key="pause_print",
+        name="Pause Print",
+        action_fn=lambda _elegoo_printer_client: _elegoo_printer_client.print_pause(),
+        icon="mdi:pause",
+        available_fn=lambda _elegoo_printer_client: _elegoo_printer_client.printer_data.status.current_status
+        == ElegooMachineStatus.PRINTING,
+    ),
```

Pylint (3.3.7)

[error] Unexpected keyword argument 'key' in constructor call

(E1123)